include/nbl/builtin/hlsl/workgroup/fft.hlsl

```diff
@@ -103,6 +107,197 @@ void FFT(NBL_REF_ARG(Accessor) sharedmemAccessor, NBL_REF_ARG(complex_t<Scalar>)
     }
 }
 
+*/
+
+// ----------------------------------------- ABOVE MARKED FOR DELETION -----------------------------------------------------
+
+template<uint16_t K, bool Inverse, typneame Scalar, class device_capabilities=void>
+struct FFT;
```

Comment:
Move `K` to AFTER device_capabilities, and default it to 2

Otherwise user has to specify its radix=2 every time, which they probably don't want (only inverse vs. forward)

Also don't call it `K` call it `Radix` cause otherwise people can get confused what the `K` is